sip-core/src/main/java/de/ikor/sip/foundation/core/translate/logging/TranslateMessageLayout.java

```diff
@@ -4,40 +4,21 @@
 import ch.qos.logback.classic.spi.ILoggingEvent;
 import ch.qos.logback.classic.spi.LoggingEvent;
 import ch.qos.logback.core.Layout;
-import de.ikor.sip.foundation.core.translate.SIPTranslateMessageService;
-import org.mapstruct.factory.Mappers;
+import de.ikor.sip.foundation.core.util.LoggingEventMessageEnricher;
 
 /**
  * Extends PatternLayout in order to keep logging pattern feature while trying to translate message
- * at the same time. In case that messageService bean is not available (ie. spring context is not
- * loaded yet) or that message key is not found, message will be logged untranslated.
+ * at the same time.
  *
  * @param <E> event type
  */
 @SuppressWarnings("java:S2326")
 public class TranslateMessageLayout<E> extends PatternLayout implements Layout<ILoggingEvent> {
-  private SIPTranslateMessageService messageService;
 
   @Override
   public String doLayout(ILoggingEvent event) {
-    messageService = initMessageService();
-    LoggingEvent cloneEvent = Mappers.getMapper(EventLogCloner.class).mapWithNoMessage(event);
-    String translatedMessage;
-    if (messageService == null) {
-      translatedMessage = event.getMessage();
-    } else {
-      translatedMessage =
-          messageService.getTranslatedMessage(event.getMessage(), event.getArgumentArray());
-    }
-
-    cloneEvent.setMessage(translatedMessage);
+    LoggingEvent cloneEvent = LoggingEventMessageEnricher.enrich(event);
```

Comment:
I see no benefit of hiding this piece of logic behind static method. Even so, I like the small refactoring done with messageService.